### DISCUSSION TITLE
Translation to Hindi

### DISCUSSION THREAD

**Discussion by paladique** [microsoft/Data-Science-For-Beginners#148] [General]
This is the discussion thread for those who are helping with this translation to divide up the work, as well as review. This is Issue #146

**Comment by subh-chaturvedi** [comment]
I can start with 1-Introduction/02-ethics.

**Comment by subh-chaturvedi** [comment]
@AdityaGarg00 can you take a look at [this hindi translation](https://github.com/subh-chaturvedi/Data-Science-For-Beginners/blob/main/1-Introduction/02-ethics/translations/README.hi.md). I will proceed and do the whole document once you give me a confirmation for this part.

**Reply by AdityaGarg00** [reply]
Great work @subh-chaturvedi  
Mostly the translations are good and easily understandable. But can you please look at the last two para's of the translation and see if those can be further improved

**Reply by subh-chaturvedi** [reply]
cool, on it

**Comment by joshi-kaushal** [comment]
@paladique please provide the main English document from which I am supposed to translate.

**Reply by jlooper** [reply]
hi @joshi-kaushal all the documents in this repo are candidates for translation. They are listed in #146 and there's a [Translations](https://github.com/microsoft/Data-Science-For-Beginners/blob/main/TRANSLATIONS.md) guide as well

**Comment by AdityaGarg00** [comment]
Hey Everyone,

I was thinking to tick the boxes in #146 whenever we start translating a particular file so as to avoid multiple people working on the same file.

**Reply by subh-chaturvedi** [reply]
cool

**Comment by joshi-kaushal** [comment]
I can start with [defining data](https://github.com/microsoft/Data-Science-For-Beginners/tree/main/1-Introduction/03-defining-data).
Lecture 01, Topic 3

**Comment by imsushant12** [comment]
hi, @AdityaGarg00 can I start with :  Vizualization base README -Quantities README ?

**Reply by AdityaGarg00** [reply]
Please Go a head @imsushant12 Just check the box right next to whichever file you are contributing in issue #146  to so that others can know that you are contributing to that particular file .

**Reply by imsushant12** [reply]
Okay, thankyou sir.

**Reply by imsushant12** [reply]
Hi @AdityaGarg00 , please review PR : #189

**Comment by Heril18** [comment]
Can I start with [4-Data-Science-Lifecycle: README?](https://github.com/microsoft/Data-Science-For-Beginners/blob/main/4-Data-Science-Lifecycle/README.md#the-data-science-lifecycle)

**Reply by AdityaGarg00** [reply]
yea sure @Heril18 You can translate any file which haven't been translated yet just check out the box next to the same in issue #146

**Comment by dhruvkrishnavaid** [comment]
Hi @AdityaGarg00, can I start with [5-Data-Science-In-Cloud base README](https://github.com/microsoft/Data-Science-For-Beginners/tree/main/5-Data-Science-In-Cloud/README.md) ?

**Reply by AdityaGarg00** [reply]
Yea Sure Go ahead @dhruvkrishnavaid

**Reply by dhruvkrishnavaid** [reply]
@AdityaGarg00 Please take a look at [this](https://github.com/dhruvkrishnavaid/Data-Science-For-Beginners/blob/main/5-Data-Science-In-Cloud/translations/README.hi.md).

**Reply by AdityaGarg00** [reply]
Do one thing make it as a pull request and add me as a reviewer it will be better

**Reply by dhruvkrishnavaid** [reply]
I've already made a pull request. See #186.

**Reply by AdityaGarg00** [reply]
ok ill review that

**Reply by AdityaGarg00** [reply]
Your translation was really great and easy to understand

**Reply by dhruvkrishnavaid** [reply]
Thanks :smile:

**Reply by dhruvkrishnavaid** [reply]
@AdityaGarg00 Please review #187 too.

**Comment by subh-chaturvedi** [comment]
made a [PR](https://github.com/microsoft/Data-Science-For-Beginners/pull/184) for the ethics readme, but there is a "Merging is blocked" error. Does it concern me, and do I need to rectify something on my side?

**Reply by jlooper** [reply]
hi, you might have a conflict, can you pull a fresh copy, add your changes, and retry the PR?

**Reply by subh-chaturvedi** [reply]
done that. Please have a look [here](https://github.com/microsoft/Data-Science-For-Beginners/pull/185)

**Reply by dhruvkrishnavaid** [reply]
@subh-chaturvedi It doesn't concern you. 
The message says: 

> The base branch restricts merging to authorized users. 

It means only the people of the organization can merge this PR i.e. only the authorised microsoft accounts can merge the PR.
So, you don't need to do anything from your side.

**Reply by subh-chaturvedi** [reply]
oh cool, thnx :)

**Comment by joshi-kaushal** [comment]
Can anyone help me to translate this sentence? Its from [unstructured data](https://github.com/joshi-kaushal/Data-Science-For-Beginners/tree/main/1-Introduction/03-defining-data#unstructured-data) section of lecture 1 topic 3.

> For example, a scientist who wants to find the average temperature of the previous month from the sensors data, but discovers that the sensor recorded an "e" in some of its recorded data to note that it was broken instead of a typical number, which means the data is incomplete.

I have translated it like this but I guess it can be improved:
>जैसे की, एक वैज्ञानिक जिसे सेंसर के डेटा से पिछले महीने के तापमान का औसत ढूंढ़ना हो, और उसे पता चलता है की सेंसर ने कुछ जगह आधेअधूरे डेटा को दर्ज करने के लिए आम क्रमांक के बजाय 'e' दर्ज किया है, जिसका मतलब है की डेटा अपूर्ण स्वरुप मैं है।

**Reply by AdityaGarg00** [reply]
I guess we can try ( जैसा की एक वैज्ञानिक है, जो सेंसर के डेटा से पिछले महीने के डेटा का औसत खोजना चाहता है मगर वो देखता है की सेंसर ने कुछ जगा एक नंबर की जंग पे "e" आउटपुट लिखी है जो खराब हुए डेटा को प्रतिनिधि करती है । जिसका मतलब हुआ की डेटा अभी अधूरा है )
there might be some gramatical errors as i have written it in hurry 😅😅

**Reply by joshi-kaushal** [reply]
yes I'll refine and use yours, thanks

**Comment by mdrpanwar** [comment]
Hey @AdityaGarg00, 
Please confirm if [Vizualization base README](https://github.com/microsoft/Data-Science-For-Beginners/tree/main/3-Data-Visualization) is still open and unpicked? I would like to work on it.

**Reply by imsushant12** [reply]
Hey @mdrpanwar , I am working on Vizualization base README!

**Reply by mdrpanwar** [reply]
@imsushant12 Thanks for informing.

**Comment by anirudhb11** [comment]
Hi @AdityaGarg00,
Could you please confirm if [6-Data-Science-In-Wild README](https://github.com/microsoft/Data-Science-For-Beginners/blob/main/6-Data-Science-In-Wild/README.md) is still open and unpicked? I would like to work on it.

**Reply by paladique** [reply]
@anirudhb11 this is still open! Feel free to select it - let us know, as @imsushant12 is interested in this as well.

**Comment by imsushant12** [comment]
Hi, @AdityaGarg00 , Can I work on Data Science in the Wild base README as no nody is currently working on it?

**Reply by paladique** [reply]
@imsushant12 if @anirudhb11 doesn't get back to us in a few days I would say go ahead and take it.

**Reply by anirudhb11** [reply]
@imsushant12  @paladique I haven't started working on this, please feel free to pick it

**Reply by imsushant12** [reply]
Sure thankyou!🙌

**Comment by paladique** [comment]
Hi all! Can someone give a thumbs up on this translation? Thank you!

https://github.com/microsoft/Data-Science-For-Beginners/pull/187

**Reply by paladique** [reply]
@imsushant12, thank you!

**Comment by paladique** [comment]
Hello! Can we get a thumbs up on this PR? Thanks!

https://github.com/microsoft/Data-Science-For-Beginners/pull/189

**Comment by paladique** [comment]
Could we get thumbs up on #201 and #202? Thank you!

**Comment by DhanyaHegde01** [comment]
Please review #207 and #208

**Comment by jlooper** [comment]
Hello everyone, could we get a linguistic review of these files: https://github.com/microsoft/Data-Science-For-Beginners/pull/360

**Comment by vedant-z** [comment]
Which files are left out to be translated ?
So that I can work on it...

**Reply by vedant-z** [reply]
@AdityaGarg00 @jlooper

**Reply by priyansh2998** [reply]
See the issue [#146 ](https://github.com/microsoft/Data-Science-For-Beginners/issues/146#issue-1013340040) itself the unchecked ones are not translated yet
